tech-spec.md

```diff
@@ -0,0 +1,31 @@
+# Tech Spec
+![overview](https://i.imgur.com/hbz9N5L.png)
+
+## Components
+
+### React
+Providing the dynamic functionality for our project showcase.
+Handles all form submissions.
+* Project Showcase page showing all projects
+* Individual project detail page for each project
+* Volunteer form on each individual project
+  - Store info in local storage.
+
+### Nginx
+Serving the static Gatsby pages.
+
+### Node server
+* Express for routing/api.
+* Typescript is the language choice.
+* Send email using Gmail API.
```

Comment:
I think it'll actually be the google cloud API. Sounds like the same thing but it isn't.